syntax/std_encoding.go

```diff
@@ -2,8 +2,28 @@ package syntax
 
 import (
 	"github.com/arr-ai/arrai/rel"
+	"github.com/arr-ai/arrai/translate/pb"
 )
 
 func stdEncoding() rel.Attr {
-	return rel.NewTupleAttr("encoding", stdEncodingJSON())
+	return rel.NewTupleAttr("encoding",
+		stdEncodingJSON(),
+		stdEncodingProtoProto(),
+		stdEncodingProto(),
+	)
+}
+
+// this is a placeholder to represent //encoding.proto.proto
```

Comment:
Nit: explain here why it's a placeholder (i.e. it simply indicates to the decode function to decode a message to the standard proto format, or something)